modules/core/test/v2/unit/coins/algo.ts

```diff
@@ -0,0 +1,68 @@
+import * as crypto from 'crypto';
+import * as algosdk from 'algosdk';
+
+const TestV2BitGo = require('../../../lib/test_bitgo');
+
+describe('ALGO:', function() {
+  let bitgo;
+  let basecoin;
+
+  before(function() {
+    bitgo = new TestV2BitGo({ env: 'test' });
```

Comment:
`env: 'mock'` in unit tests, so it's very obvious when a nock is missing

Reply:
done